modules/styling/stories/Stencils.stories.mdx

```diff
@@ -0,0 +1,270 @@
+import {SymbolDoc} from '@workday/canvas-kit-docs';
+
+import {CreateStencil} from './examples/CreateStencil';
+
+<Meta title="Features/Styling/Stencils" />
+
+## Stencil
+
+Stencils are a reusable function that returns styles based on modifiers passed in. A Stencil should
```

Comment:
does it return styles or a class name with your styles ?

Reply:
It returns `{ style: CSSObject, className: string}`